### PR TITLE
document order of svdvals

### LINKED ISSUE #368: State explicitly in the documentation that `svd` and related functions return singular values in descending order
In all cases I've tried, `svd` returns singular values sorted in decreasing order. If this is guaranteed, it should be stated explicitly in the documentation.

### LINKED ISSUE #18482: State explicitly in the documentation that `svd` and related functions return singular values in descending order
In all cases I've tried, `svd` returns singular values sorted in decreasing order. If this is guaranteed, it should be stated explicitly in the documentation.

### PR DESCRIPTION
**Pull request by fredrikekre**
fix JuliaLang/LinearAlgebra.jl#368

ref: http://www.netlib.org/lapack/explore-html/d1/d7e/group__double_g_esing_ga76f797b6a9e278ad7b21aae2b4a55d76.html#ga76f797b6a9e278ad7b21aae2b4a55d76

### CODE REVIEW

File: base/linalg/svd.jl [hunk 1]
```diff
@@ -15,10 +15,6 @@ SVD(U::AbstractArray{T}, S::Vector{Tr}, Vt::AbstractArray{T}) where {T,Tr} = SVD
 
 `svdfact!` is the same as [`svdfact`](@ref), but saves space by
 overwriting the input `A`, instead of creating a copy.
-
-If `thin=true` (default), a thin SVD is returned. For a ``M \\times N`` matrix
```
Comment (fredrikekre) [inline base/linalg/svd.jl]: I removed this, this is consistent with all the other `!` functions here, they refer to the non `!` function.

File: base/linalg/svd.jl [hunk 2]
```diff
@@ -74,7 +71,7 @@ svdfact(x::Integer; thin::Bool=true) = svdfact(float(x), thin=thin)
     svd(A, thin::Bool=true) -> U, S, V
 
 Computes the SVD of `A`, returning `U`, vector `S`, and `V` such that
-`A == U*diagm(S)*V'`.
+`A == U*diagm(S)*V'`. The singular values in `S` is sorted in descending order.
```
Comment (kshyatt) [inline base/linalg/svd.jl]: "are sorted"

**Review by kshyatt — APPROVED** [review]
modulo typo lgtm

File: base/linalg/svd.jl (hunk 2 above)
Comment (fredrikekre) [inline base/linalg/svd.jl]: thanks

**Review by Sacha0 — APPROVED** [review]
Thanks @fredrikekre ! :)

**Pull request merged**